### PR TITLE
[dwdunwetter] documentation for binding updated 

### PR DESCRIPTION
**Pull request by jschanz**

### CODE REVIEW

File: bundles/org.openhab.binding.dwdunwetter/README.md [hunk 1]
```diff
@@ -114,3 +132,29 @@ then
 end 
 
 ```
+
+dwdunwetter_severity.map
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.dwdunwetter/README.md:136]: Please rename these transformation files to `dwdunwetter_severity_de.map` as it contains German mappings.

File: bundles/org.openhab.binding.dwdunwetter/README.md [hunk 2]
```diff
@@ -114,3 +132,29 @@ then
...
+Minor=Wetterwarnung
+Moderate=Markante Wetterwarnung
+Severe=Unwetterwarnung
+Extreme=Extreme Unwetterwarnung
+NULL=undefiniert
+```
+
+
+
+dwdunwetter_urgency.map
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.dwdunwetter/README.md:148]: Please rename these transformation files to `dwdunwetter_urgency_de.map` as it contains German mappings.

File: bundles/org.openhab.binding.dwdunwetter/README.md [hunk 3]
```diff
@@ -114,3 +132,29 @@ then
...
 ```
+
+dwdunwetter_severity.map
+
+```
+Minor=Wetterwarnung
+Moderate=Markante Wetterwarnung
+Severe=Unwetterwarnung
+Extreme=Extreme Unwetterwarnung
+NULL=undefiniert
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.dwdunwetter/README.md:143]: We should amend a mapping for `UNDEF` too:

```suggestion
NULL=undefiniert
UNDEF=undefiniert
```

File: bundles/org.openhab.binding.dwdunwetter/README.md [hunk 4]
```diff
@@ -114,3 +132,29 @@ then
...
+```
+
+
+
+dwdunwetter_urgency.map
+
+```
+Immediate=Warnung
+Future=Vorabinformation
+NULL=undefiniert
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.dwdunwetter/README.md:153]:
```suggestion
NULL=undefiniert
UNDEF=undefiniert
```

File: bundles/org.openhab.binding.dwdunwetter/README.md [hunk 5]
```diff
@@ -63,32 +63,50 @@ For rules that need to fire if a new warning occurs, there is the trigger channe
...
-String WarningCologneGueltigAb "Valid from [%s]" { channel="dwdunwetter:dwdwarnings:cologne:onset1" }
-String WarningCologneGueltigBis "Valid to [%s]" { channel="dwdunwetter:dwdwarnings:cologne:expires1" }
-String WarningCologneTyp "Type [%s]" { channel="dwdunwetter:dwdwarnings:cologne:event1" }
-String WarningCologneTitel "[%s]" { channel="dwdunwetter:dwdwarnings:cologne:headline1" }
-String WarningCologneHoeheAb "Height from [%d m]" { channel="dwdunwetter:dwdwarnings:cologne:altitude1" }
-String WarningCologneHoeheBis "Height to [%d m]" { channel="dwdunwetter:dwdwarnings:cologne:ceiling1" }
-String WarningCologneUrgency "[%s]" { channel="dwdunwetter:dwdwarnings:cologne:urgency1" }
-String WarningCologneInstruction "Additional information: [%s]" { channel="dwdunwetter:dwdwarnings:cologne:instruction1" }
+Switch        WarningCologne_1             "Weather warning"                             { channel="dwdunwetter:dwdwarnings:cologne:warning1" }
+String        WarningCologneServerity_1    "Severity [MAP(dwdunwetter_severity.map):%s]" { channel="dwdunwetter:dwdwarnings:cologne:severity1" }
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.dwdunwetter/README.md:93]: Rename mapping file.

File: bundles/org.openhab.binding.dwdunwetter/README.md [hunk 6]
```diff
@@ -63,32 +63,50 @@ For rules that need to fire if a new warning occurs, there is the trigger channe
...
+DateTime      WarningCologneAusgabedatum_1 "Issued at [%s]"                              { channel="dwdunwetter:dwdwarnings:cologne:effective1" }
+DateTime      WarningCologneGueltigAb_1    "Valid from [%s]"                             { channel="dwdunwetter:dwdwarnings:cologne:onset1" }
+DateTime      WarningCologneGueltigBis_1   "Valid to [%s]"                               { channel="dwdunwetter:dwdwarnings:cologne:expires1" }
+String        WarningCologneTyp_1          "Event [%s]"                                  { channel="dwdunwetter:dwdwarnings:cologne:event1" }
+String        WarningCologneTitel_1        "[%s]"                                        { channel="dwdunwetter:dwdwarnings:cologne:headline1" }
+Number:Length WarningCologneHoeheAb_1      "Height from [%d m]"                          { channel="dwdunwetter:dwdwarnings:cologne:altitude1" }
+Number:Length WarningCologneHoeheBis_1     "Height to [%d m]"                            { channel="dwdunwetter:dwdwarnings:cologne:ceiling1" }
+String        WarningCologneUrgency_1      "[MAP(dwdunwetter_urgency.map):%s]"           { channel="dwdunwetter:dwdwarnings:cologne:urgency1" }
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.dwdunwetter/README.md:102]: same here

File: bundles/org.openhab.binding.dwdunwetter/README.md [hunk 7]
```diff
@@ -63,32 +63,50 @@ For rules that need to fire if a new warning occurs, there is the trigger channe
...
+String        WarningCologneInstruction_1  "Additional information: [%s]"                { channel="dwdunwetter:dwdwarnings:cologne:instruction1" }
+
+Switch        WarningCologne_2             "Weather warning"                             { channel="dwdunwetter:dwdwarnings:cologne:warning2" }
+String        WarningCologneServerity_2    "Severity [MAP(dwdunwetter_severity.map):%s]" { channel="dwdunwetter:dwdwarnings:cologne:severity2" }
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.dwdunwetter/README.md:106]: same here

**Review by cweitkamp — CHANGES_REQUESTED** [review]
Thanks for the update. I left some comments.

File: bundles/org.openhab.binding.dwdunwetter/README.md (hunk 1 above)
Comment (jschanz) [reply bundles/org.openhab.binding.dwdunwetter/README.md]: Thanks... Changed and fixed

File: bundles/org.openhab.binding.dwdunwetter/README.md (hunk 2 above)
Comment (jschanz) [reply bundles/org.openhab.binding.dwdunwetter/README.md]: Thanks... Changed and fixed

File: bundles/org.openhab.binding.dwdunwetter/README.md (hunk 3 above)
Comment (jschanz) [reply bundles/org.openhab.binding.dwdunwetter/README.md:150]: Thanks... Changed and fixed

File: bundles/org.openhab.binding.dwdunwetter/README.md (hunk 4 above)
Comment (jschanz) [reply bundles/org.openhab.binding.dwdunwetter/README.md:161]: Thanks... Changed and fixed

File: bundles/org.openhab.binding.dwdunwetter/README.md (hunk 6 above)
Comment (jschanz) [reply bundles/org.openhab.binding.dwdunwetter/README.md]: Thanks... Changed and fixed

File: bundles/org.openhab.binding.dwdunwetter/README.md (hunk 7 above)
Comment (jschanz) [reply bundles/org.openhab.binding.dwdunwetter/README.md]: Thanks... Changed and fixed

File: bundles/org.openhab.binding.dwdunwetter/README.md (hunk 5 above)
Comment (jschanz) [reply bundles/org.openhab.binding.dwdunwetter/README.md]: Thanks... Changed and fixed

**Review by cweitkamp — APPROVED** [review]
LGTM. Thanks.

Merging this with small path exception this time. Please remember to [sign-your-work](https://www.openhab.org/docs/developer/contributing.html#sign-your-work) in the future. Thanks.

**Pull request merged**